src/file/file.service.spec.ts

```diff
@@ -489,6 +489,294 @@ describe('File Logic', () => {
 
   });
 
+  describe('#createShortcut', () => {
+    // create file section
+
+    it('should throw an error when key is not sent with file', async () => {
+      await FileService.create(bucket, 'myFolder', USER.id, 'Text', 'drive', '', '', size)
+        .should.eventually.be.rejectedWith(ServerError, 'No key sent');
+    });
+
+    it('should not throw an error if key is not sent with a folder', async () => {
+      await FileService.create(bucket, 'myFolder', USER.id, FolderContentType, 'drive').should.eventually.exist;
+    });
+
+    it('should throw error: same owner, folder, appID and filename', async () => {
+      await FileService.create(bucket, 'myFile', USER.id, 'Text', 'drive', null, KEY, size).should.eventually.exist;
+      await FileService.create(bucket, 'myFile', USER.id, 'Other', 'drive', null, KEY2, size)
+        .should.eventually.be.rejectedWith(FileExistsWithSameName);
+    });
+
+    it('should not throw error: same folder and filename, different owner', async () => {
+      await FileService.create(bucket, 'myFile', USER.id, 'Text', 'drive', null, KEY, size).should.eventually.exist;
+      await FileService.create(bucket, 'myFile', '654321', 'Other', 'drive', null, KEY2, size).should.eventually.exist;
+    });
+
+    it('should not throw error: same folder and filename, different appID', async () => {
+      await FileService.create(bucket, 'myFile', USER.id, 'Text', 'drive', null, KEY, size).should.eventually.exist;
+      await FileService.create(bucket, 'myFile', USER.id, 'Other', 'dropbox', null, KEY2, size).should.eventually.exist;
+    });
+
+    it('should throw error: parent app id is not the same', async () => {
+      const parent = await FileService.create(bucket, 'myFile', USER.id, 'Text', 'drive', null, KEY, size).should.eventually.exist;
+      await FileService.create(bucket, 'myFile', USER.id, 'Other', 'dropbox', parent.id, KEY2, size)
+        .should.eventually.be.rejectedWith(FileParentAppIDNotEqual);
+    });
+
+    it('should create file when parent app id is the same', async () => {
+      const parent = await FileService.create(bucket, 'myFile', USER.id, 'Text', 'drive', null, KEY, size).should.eventually.exist;
+      await FileService.create(bucket, 'myFile', USER.id, 'Other', 'drive', parent.id, KEY2, size).should.eventually.exist;
+    });
```

Comment:
You just copy pasted from the regular file but it is incorrect. Please add some thoughts on the test you are trying to run

Reply:
okay

Reply:
deleted it, done